src/fio/fioSchema.js

```diff
@@ -2,6 +2,32 @@
 
 import { asArray, asNumber, asObject, asOptional, asString } from 'cleaners'
 
+export const asGetFioName = asObject({
+  fio_domains: asArray(
+    asObject({
+      fio_domain: asString,
+      expiration: asString,
+      is_public: asNumber
+    })
+  ),
+  fio_addresses: asArray(
+    asObject({
+      fio_address: asString,
+      expiration: asString
+    })
+  )
+})
+
+type GetFioName = $Call<typeof asGetFioName>
+
+export function fioGetNameChecker(result: GetFioName): string | void {
+  let value
+  try {
+    value = JSON.stringify(asGetFioName(result))
+  } catch (e) {}
+  return value
```

Comment:
No need for the `value` var:

```ts
try {
  return JSON.stringify(asgetFioName(result))
} catch (e) {}
```

Also, I'm wary of the exception handling here. Are we okay with silencing all errors here; both stringify exceptions and the cleaner exception?